<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SPGRASD11" name="com.ats.qa.testcases.Registeration.RegisterationTest" tests="15" failures="0" timestamp="3 Sep 2020 04:16:27 GMT" time="38.194" errors="3">
  <testcase name="Registeration_clickonNewwalkinEntry" time="2.159" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonPan" time="4.212" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonsubmit" time="4.098" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonName" time="0.309" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonLocation" time="7.075" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonDetails" time="2.119" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonsavebutton" time="0.002" classname="com.ats.qa.testcases.Registeration.RegisterationTest">
    <error type="java.lang.Throwable" message="Method RegisterationTest.Registeration_clickonsavebutton()[pri:0, instance:com.ats.qa.testcases.Registeration.RegisterationTest@cc285f4] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method RegisterationTest.Registeration_clickonsavebutton()[pri:0, instance:com.ats.qa.testcases.Registeration.RegisterationTest@cc285f4] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Registeration_clickonsavebutton -->
  <testcase name="Registeration_clickOnAdharcard" time="2.181" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonImpactsource" time="0.175" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonRegisterationstatus" time="0.000" classname="com.ats.qa.testcases.Registeration.RegisterationTest">
    <error type="java.lang.Throwable" message="Method RegisterationTest.Registeration_clickonRegisterationstatus()[pri:0, instance:com.ats.qa.testcases.Registeration.RegisterationTest@cc285f4] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method RegisterationTest.Registeration_clickonRegisterationstatus()[pri:0, instance:com.ats.qa.testcases.Registeration.RegisterationTest@cc285f4] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1037)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Registeration_clickonRegisterationstatus -->
  <testcase name="Registeration_clickonRegister" time="7.901" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonCalender" time="2.484" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonCategory" time="0.168" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonMobileno" time="0.267" classname="com.ats.qa.testcases.Registeration.RegisterationTest"/>
  <testcase name="Registeration_clickonduplicatecheck" time="5.044" classname="com.ats.qa.testcases.Registeration.RegisterationTest">
    <error type="org.openqa.selenium.UnhandledAlertException" message="unexpected alert open: {Alert text : Cannot Register}
  (Session info: chrome=80.0.3987.106): Cannot Register
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:25:48&#039;
System info: host: &#039;SPGRASD11&#039;, ip: &#039;172.27.85.33&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_152&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 80.0.3987.106, chrome: {chromedriverVersion: 80.0.3987.16 (320f6526c1632..., userDataDir: C:\Users\RCHAVA~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:56125}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 0b5d9789bf07858db98d7720a7bf7241">
      <![CDATA[org.openqa.selenium.UnhandledAlertException: unexpected alert open: {Alert text : Cannot Register}
  (Session info: chrome=80.0.3987.106): Cannot Register
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:48'
System info: host: 'SPGRASD11', ip: '172.27.85.33', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_152'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 80.0.3987.106, chrome: {chromedriverVersion: 80.0.3987.16 (320f6526c1632..., userDataDir: C:\Users\RCHAVA~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:56125}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 0b5d9789bf07858db98d7720a7bf7241
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:120)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.executeScript(RemoteWebDriver.java:489)
	at com.ats.qa.testcases.Registeration.RegisterationTest.Registeration_clickonduplicatecheck(RegisterationTest.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Registeration_clickonduplicatecheck -->
</testsuite> <!-- com.ats.qa.testcases.Registeration.RegisterationTest -->
